connexion/json_schema.py

```diff
@@ -128,6 +128,29 @@ def nullable_validation_fn(validator, to_validate, instance, schema):
     return nullable_validation_fn
 
 
+def validate_required(validator, required, instance, schema):
+    if not validator.is_type(instance, "object"):
```

Comment:
Please add a hint for the return type, also possibly a Pydoc comment, maybe:

    """Check a request for required fields, ignoring fields marked readOnly"""